### ISSUE TITLE
Questions about the experimental results of other models in the paper.

### ISSUE THREAD

**Issue by POWY3** [zizheng-guo/RhythmFormer#8]
Thank you ever so much for exposing your code, this is very helpful for me. After I successfully reproduced your method on the PURE dataset, I attempted to obtain the experimental results of other models in your paper such as PhysNet on the PURE dataset. I tried to make some alterations to the yaml file, but I still can't obtain the results shared in your article. Could you share the specific experimental design?

**Comment by zizheng-guo** [comment]
Thank you for your interest in our work. For the intra-dataset evaluation, the results of other models are sourced from their original papers—or from follow-up studies if unavailable in the original publications. I haven't run these experiments myself. Would you be willing to share your findings?

**Comment by POWY3** [comment]
I used the same hyperparameters as for training Rhythmformer, only in the preprocessing parameters I changed "Standardized" to "DiffNormalized" which is more suitable for Physnet. This is the result of USE_LAST_EPOCH being False.
![Image](https://github.com/user-attachments/assets/9806180e-15a1-47f5-bbb7-e962cd55c95a)
That is the result of USE_LAST_EPOCH being True
![Image](https://github.com/user-attachments/assets/c9327787-4692-48be-b08d-f3673b8a0f3e)
So I don't know which result to refer to。

**Comment by POWY3** [comment]
> I used the same hyperparameters as for training Rhythmformer, only in the preprocessing parameters I changed "Standardized" to "DiffNormalized" which is more suitable for Physnet. This is the result of USE_LAST_EPOCH being False.我使用了与训练 Rhythmformer 相同的超参数，只是在预处理参数中，我将 “Standardized” 更改为 “DiffNormalized”，这更适合 Physnet。这是 USE_LAST_EPOCH 为 False 的结果。 ![Image](https://github.com/user-attachments/assets/9806180e-15a1-47f5-bbb7-e962cd55c95a) That is the result of USE_LAST_EPOCH being True这就是 USE_LAST_EPOCH 为 True 的结果 ![Image](https://github.com/user-attachments/assets/c9327787-4692-48be-b08d-f3673b8a0f3e) So I don't know which result to refer to。所以我不知道该引用哪个结果。

By the way i did not use the AUG

**Comment by zizheng-guo** [comment]
The test protocol of PURE does not have a validation set, so USE_LAST_EPOCH should be set to True. The reason why True performs better might be that although a checkpoint with better pulse waves is selected, the HR  may not necessarily be better.

**Comment by POWY3** [comment]
Thank you for your reply! But i have another question, the result of the experiment with the rppg-toolbox environment is much better than the one in the paper. Is it possible that the rppg-toolbox environment has made some optimizations to it.
